src/actors/builtin/cron/cron_actor.go

```diff
@@ -1,11 +1,28 @@
 package cron
 
 import (
+	addr "github.com/filecoin-project/go-address"
 	abi "github.com/filecoin-project/specs/actors/abi"
 	builtin "github.com/filecoin-project/specs/actors/builtin"
 	vmr "github.com/filecoin-project/specs/actors/runtime"
 )
 
+type CronActorState interface {
+	Impl() *CronActorState_I
+}
+
+type CronTableEntry interface {
+	ToAddr() addr.Address
+	MethodNum() abi.MethodNum
+	Impl() *CronTableEntry_I
+}
```

Comment:
This is too faithful a translation of the generated code :-) Please simplify these down to vanilla structs with exported fields and no methods.